### PR TITLE
Feature/tuya panel gateway sdk

### PR DESCRIPTION
**Pull request by Oliverzzq**
## Description

Universal gateway control business component
1.AddDeviceTip
2.SetPassword
3.TempHumWithBlur
4.TopBarWithArc

## Package

- [ ] tuya-panel-animation-sdk
- [ ] tuya-panel-api
- [ ] tuya-panel-electrician-sdk
- [ ] tuya-panel-ipc-sdk
- [ ] tuya-panel-lamp-sdk
- [ ] tuya-panel-lock-sdk
- [ ] tuya-panel-robot-sdk
- [x] tuya-panel-gateway-sdk

## Type of change

Please select the relevant option.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -38,24 +38,26 @@
     "typescript": "^4.0.5"
   },
   "scripts": {
-    "version:pre": "yarn build && lerna version --canary --yes --pre-dist-tag=rc --preid=rc",
```
Comment (youngjuning) [inline package.json:41]: 这里不需要改，提升版本和发布是分开的

File: package.json [hunk 2]
```diff
@@ -38,24 +38,26 @@
...
-    "version:prepatch": "yarn build && lerna version --canary patch --yes --pre-dist-tag=rc --preid=rc",
```
Comment (youngjuning) [inline package.json:42]: 这里不要改动

**Review by youngjuning — CHANGES_REQUESTED** [review]

**Comment by youngjuning** [comment]
@Oliverzzq 大佬，看下需要改的地方

**Review by youngjuning — APPROVED** [review]
LGTM~

File: packages/tuya-panel-gateway-sdk/src/components/addDeviceTip/i18n/index.ts [hunk 3]
```diff
@@ -0,0 +1,6 @@
+import { Strings } from 'tuya-panel-kit';
+import LocalStrings from './strings';
+
+Strings.applyStrings(LocalStrings);
```
Comment (ShinyLeee) [inline packages/tuya-panel-gateway-sdk/src/components/addDeviceTip/i18n/index.ts:4]: 独立实例化一个 I18N，建议不要复用 components 的 Strings，key 可能会重复

File: packages/tuya-panel-gateway-sdk/src/components/setPassword/i18n/index.ts [hunk 4]
```diff
@@ -0,0 +1,6 @@
+import { Strings } from 'tuya-panel-kit';
+import LocalStrings from './strings';
+
+Strings.applyStrings(LocalStrings);
```
Comment (ShinyLeee) [inline packages/tuya-panel-gateway-sdk/src/components/setPassword/i18n/index.ts:4]: 独立实例化一个 I18N，建议不要复用 components 的 Strings，key 可能会重复

**Review by ShinyLeee — APPROVED** [review]

**Pull request merged**